.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.4
```

Comment:
In that case if existing master is stable enough then we should first tag it and then bump version after merging this change while updating core dependency in composer.json